lib/test/t_TruncatedNormal_std.expout

```diff
@@ -17,24 +17,24 @@ mean=class=Point name=Unnamed dimension=1 values=[0.0635542]
 covariance=class=CovarianceMatrix dimension=1 implementation=class=MatrixImplementation name=Unnamed rows=1 columns=1 values=[1.25177]
 Kolmogorov test for the generator, sample size=100 is accepted
 Kolmogorov test for the generator, sample size=1000 is accepted
-Point= class=Point name=Unnamed dimension=1 values=[1]
-ddf     =class=Point name=Unnamed dimension=1 values=[-0.0148955]
+Point= class=Point name=Unnamed dimension=1 values=[0]
+ddf     =class=Point name=Unnamed dimension=1 values=[0.0148955]
 log pdf=-1.31632
 pdf     =0.26812
 pdf (FD)=0.26812
-cdf=0.743877
-ccdf=0.256123
-survival=0.256123
+cdf=0.47326
+ccdf=0.52674
+survival=0.52674
```

Comment:
These are huge changes!

Reply:
@josephmure This is because I had to change the bounds of the "usual" case in order to have an overlap between the support of the usual case example and the exceptional case example, and I also changed the point at which these functions are computed (1.5->3.5)

Reply:
OK